packages/ember-runtime/tests/computed/reduce_computed_macros_test.js

```diff
@@ -492,6 +493,78 @@ QUnit.test('properties values can be replaced', function() {
   });
 });
 
+QUnit.module('computed.uniqBy', {
```

Comment:
This `module` and all `tests` should be wrapped in an `if (isEnabled('ember-runtime-computed-uniq-by')) { }`.